src/actions/common.go

```diff
@@ -16,15 +16,15 @@ import (
 var BehaviourMap = map[string]func(*exinfra.Infra, *exargs.Configuration, exinfra.Bricks) (int, error){
 	"clean":         Clean,
 	"help":          Help,
-	"init":          Default,
+	"init":          PassthroughAction,
```

Comment:
What do you think about :
StandardExecuteAllEvenIfFail

Reply:
many other fonction passThroughAction

Reply:
It is a mouthful, and is very unclear (to me). "Standard" here does not come to mind to me as "regular" or "normal behaviour-ed", but more as a Std(in|out|err) semantic, and the suffix `EvenIfFail`, even if somewhat clear, could be replaced more succintly with "ForceExecute", and feels a bit weird considering it can still return an error.

What I mean is that long variables / function names are considered code smell. Concise expression of behaviour (with context) would mean the opposite.

If you feel like `PassthroughAction` does not express well enough what it does, it means that the action itself has not clearly defined behavior, or that the abstraction / organization around it is not clear enough.

But what don't you find clear about this naming? What does `PasstroughAction` doesn't express that you'd like to express? Or what does it express that you'd rather it wouldn't?

Reply:
The problem is that lay, remove, plan, help can correspond to passThroughAction. Only clean show and debug* can not be called like that.

The behaviour is :
- it's an action passed to module (*)
- only zero is considered as a success (1)
- execute all bricksToExecute even if one bricks failed (1)
- used for action that don't change anything (2)
- used for action that doesn't need to create input file (2)

To find a fully discriminant name for the behaviour we have to say *+1  or *+2

Or as it's the only behaviour that some action have in common we can perhaps say StandardBehaviour or RegularBehaviour if you prefer

Reply:
Oh ok I see; but the first assumption does not ring true to me.

It is not because some other actions *can* match the same behavior, that it should be reflected it their name, or on this action's name.

They're 2 different actions for a reason. You don't name something because of how it looks like something else, you name it for what it does.

Reply:
Maybe it's a useless debat. But for me you name something to let you discriminate it from other else call it function or doSomething because it's a function. And here you name something by a description that discriminate partially the behaviour from some others it make no sense. call it bricksBehaviour.

As we failed to find a name that describe the behaviour we can find it a name. And use a suffix to say that all function have a same order aim.

perhaps we should have :
- PlanBehaviour
- LayBehaviour
- RegularBehaviour/DefaultBehaviour/OtherBehaviour
- ...
or perhaps we should have :
- PlanAction
- LayAction
- RegularAction/DefaultAction/OtherAction